src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImportAliasCommand.cs

```diff
@@ -99,6 +100,93 @@ public SwitchParameter Force
         #endregion Parameters
 
         #region Command code
+        
+        private static bool OnlyContainsWhitespace(string line)
+        {
+            bool result = true;
+
+            foreach (char c in line)
+            {
+                if (char.IsWhiteSpace(c))
+                {
+                    continue;
+                }
+
+                result = false;
+                break;
+            }
+
+            return result;
+        }
+      
+        private static Collection<string> ParseCsvLine(string csv)
+        {        
+            ReadOnlySpan<char> csvTrimmed = csv.Trim();
+            Collection<string> result = new Collection<string>();
+            StringBuilder wordBuffer = new StringBuilder();
+
+            for (int i = 0; i < csvTrimmed.Length; i++) 
+            {
+                char nextChar = csvTrimmed[i];
+
+                // if next character was delimiter or we are at the end, add string to result and clear wordBuffer
+                // else if next character was quote, perform reading until next quote and add it to wordBuffer
+                // else read and add it to wordBuffer
+                if (nextChar == ',') 
+                {
+                    result.Add(wordBuffer.ToString());
+                    wordBuffer.Clear();
+                } 
+                else if (nextChar == '"') 
+                {
+                    bool inQuotes = true;
+                    
+                    // if we are within a quote section, read and append to wordBuffer until we find a next quote that is not followed by another quote
+                    // if it is a single quote, escape the quote section
+                    // if the quote is followed by an other quote, do not escape and add a quote character to wordBuffer
+                    while (i < csvTrimmed.Length && inQuotes) 
+                    {
+                        i++;
+                        nextChar = csvTrimmed[i];
```

Comment:
Boundary after i++ can be = csvTrimmed.Length:
```c#
 while (++i < csvTrimmed.Length && inQuotes)
```

Reply:
Wouldn't that change cause `i` to increment regardless whether the while loop is entered?

Reply:
Wouldn't that introduce a bug? Replacing `i` with `++i` would increment `i` regardless whether `inQuotes` is true or false. In other words, when `i` is at the index of the second `"`, it will increment `i` one additional time, skipping the first character after we escape the quote sequence.

For example, `a"b"cd` would then be parsed to `abd`, which is incorrect as it should be parsed to `abcd`.

Reply:
@iSazonov , can you please looks at this again? thank you.

Reply:
My comment is only about that there can be boundary overflow because of `i++` without follow check.

Reply:
That would require some different solution, which I think is solved by replacing line 147 with `i + 1 < csvTrimmed.Length && inQuotes` and line 154 with `(i < csvTrimmed.Length && csvTrimmed[i + 1] == '"')`